content/attacks/posts/2021-02-13-Alpha-Finance.md

```diff
@@ -0,0 +1,50 @@
+---
+date: 2021-02-13
+categories:

```

Comment:
Rewrite headers according to [new changes](https://github.com/1712n/dn-institute#readme). In this case, `entity-types` are `DeFi` and `Yield Aggregator`, and `attack -types` are `Smart Contract Exploit` and `Flash Loan Attack`